tests/end2end/features/downloads.feature

```diff
@@ -337,6 +337,27 @@ Feature: Downloading things from a website.
         And I open the download with a placeholder
         Then "Opening *download.bin* with [*python*]" should be logged
 
+    Scenario: Opening a download with default-open-dispatcher set
+        When I set a test python default-open-dispatcher
+        And I open data/downloads/download.bin without waiting
+        And I wait until the download is finished
+        And I run :download-open
+        Then "Opening *download.bin* with [*python*]" should be logged
+
+    Scenario: Opening a download with default-open-dispatcher set to cat
```

Comment:
Interestingly enough it passed on AppVeyor, maybe it happens to have some MinGW `cat` installed. But yeah, I think it'd be better to get rid of it.